components/prow/manifests/nginx-controller.yaml

```diff
@@ -354,7 +354,7 @@ spec:
       paths:
       - backend:
           serviceName: prometheus-meta
-          servicePort: 80
+          servicePort: http
```

Comment:
Similarly, look at port names in https://github.com/prometheus/prombench/blob/a6b972a831f7eb4fe756359fc36ff12502844193/components/prombench/manifests/results/prometheus-meta.yaml#L166